src/types/check_config.rs

```diff
@@ -59,6 +59,12 @@ pub struct CheckConfig {
     /// If we should allow sampling on the trace spans.
     #[serde(default)]
     pub trace_sampling: bool,
+
+    #[serde(default)]
+    pub active_regions: Option<Vec<String>>,
```

Comment:
We'll change these from Option in the future right?

Reply:
Ideally yeah, this should always be present